dataset_utils/convert.cpp

```diff
@@ -0,0 +1,105 @@
+#include <iostream>
+#include <unordered_map>
+#include <boost/range/combine.hpp>
+#include <boost/lexical_cast.hpp>
+#include <boost/property_tree/ptree.hpp>
+#include <boost/property_tree/xml_parser.hpp>
+#include <boost/property_tree/json_parser.hpp>
+#include <boost/tokenizer.hpp>
+#include <string>
+#include <fstream>
+#include <unordered_map>
+
+
+using namespace boost::property_tree;
+using namespace boost;
+
+class Convert
+{
+  auto tokenize(std::string& line)
+  {
+    std::vector<std::string> col_names;
+    tokenizer<escaped_list_separator<char> > tk(line, escaped_list_separator<char>());
+    for (tokenizer<escaped_list_separator<char> >::iterator i(tk.begin()); i != tk.end(); ++i)
+      col_names.push_back(*i);
+    return col_names;
+  }
+
+	auto create_XML(std::vector<std::string>& tags, std::vector<std::string> rows)
```

Comment:
I think there is some issue with indentation. Also, can we replace auto with data type (makes it easier to figure out all properties).

Reply:
Can I also do this?